lisa/messages.py

```diff
@@ -262,6 +262,16 @@ class ProvisionBootTimeMessage(MessageBase):
     information: Dict[str, str] = field(default_factory=dict)
 
 
+@dataclass
+class KernelBuildMessage(MessageBase):
+    type: str = "KernelBuild"
+    old_kernel_version: str = ""
+    new_kernel_version: str = ""
+    build_sucess: bool = False
```

Comment:
It looks build/boot not used separatedly. Merge them into one result.

Reply:
Updated